graf3d/eve7/src/REveManager.cxx

```diff
@@ -187,8 +184,7 @@ REveManager::~REveManager()
    fHighlight->DecDenyDestroy();
    fSelection->DecDenyDestroy();
 
-   gROOT->GetListOfBrowsables()->Remove(fMacroFolder);
-   delete fMacroFolder;
+   for (auto i = fMethCallMap.begin(); i != fMethCallMap.end(); ++i) delete i->second;
```

Comment:
May be one should use `std::unordered_map<std::string, std::unique_ptr<TMethodCall>> fMethCallMap;` 
In such case you do not need to care about ownership